framework/Dockerfile

```diff
@@ -34,6 +34,30 @@ ENV IDRINTH_API_BENCH_VERSION=$BUILD_VERSION
 ENV IDRINTH_API_BENCH_BUILT=$BUILD_TIME
 
 RUN mkdir -p /usr/local/lib/node_modules/@idrinth/api-bench
+RUN git clone https://github.com/idrinth-api-bench/framework \
+    && npm ci \
+    && clean-modules clean --yes \
+    && mkdir -p /usr/local/lib/node_modules/@idrinth/api-bench \
+    && cp --recursive /api-bench/framework/* /usr/local/lib/node_modules/@idrinth/api-bench/ \
+    && ls -la \
```

Comment:
why are you listing the directory/folder content?

Reply:
removed ls cmd